aspnetcore/performance/response-compression.md

```diff
@@ -15,27 +15,36 @@ uid: performance/response-compression
 
 Network bandwidth is a limited resource. Reducing the size of the response usually increases the responsiveness of an app, often dramatically. One way to reduce payload sizes is to compress an app's responses.
 
+<a name="risk"></a>
+
+## Compression with HTTPS
+
+Compressed responses over secure connections can be controlled with the <xref:Microsoft.AspNetCore.ResponseCompression.ResponseCompressionOptions.EnableForHttps> option, which is disabled by default because of the security risk. Using compression with dynamically generated pages can make expose the app to [CRIME](https://wikipedia.org/wiki/CRIME_(security_exploit)) and [BREACH](https://wikipedia.org/wiki/BREACH_(security_exploit)) attacks. CRIME and BREACH attacks can be
+ mitigated in ASP.NET Core by antiforgery tokens. For more information, see <xref:security/anti-request-forgery>. For information on mitigating BREACH attacks, see [mitigations](http://www.breachattack.com/#mitigations) at http://www.breachattack.com/
+
+Even when `EnableForHttps` is disabled in the app, IIS, IIS Express, and [Azure App Service](xref:host-and-deploy/azure-iis-errors-reference) can apply gzip at the IIS web server. When reviewing response headers, take note of the [Server](https://developer.mozilla.org/docs/Web/HTTP/Headers/Server) value.
+
```

Comment:
@blowdart  review required

Reply:
When the server is IIS, gzip is used with HTTPS.  I was surprised when Gzip was used on Azure.

Reply:
@blowdart  I appended:
An unexpected `content-encoding` response header value may be the result of the server and not the application configuration.

Maybe I'm over reacting to being initially flummoxed by the default config using Gzip when deployed to Azure.